CMake/platforms/ps2.cmake

```diff
@@ -0,0 +1,41 @@
+set(NONET ON)
+set(NOSOUND OFF)
+set(DISABLE_DEMOMODE ON)
+set(ASAN OFF)
+set(UBSAN OFF)
+set(BUILD_TESTING OFF)
+
+set(DEVILUTIONX_SYSTEM_SDL2 OFF)
+set(DEVILUTIONX_SYSTEM_BZIP2 OFF)
+set(DEVILUTIONX_SYSTEM_LIBFMT OFF)
+set(DEVILUTIONX_STATIC_LIBFMT ON)
+set(PREFILL_PLAYER_NAME ON)
+set(STREAM_ALL_AUDIO ON)
+set(NOEXIT ON)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/threads-stub")
+
+set(BUILD_ASSETS_MPQ OFF)
+set(UNPACKED_MPQS ON)
+
+#set(DEVILUTIONX_STATIC_CXX_STDLIB OFF)
+
+# -fmerge-all-constants saves ~4 KiB
+#set(_extra_flags "-fmerge-all-constants")
+#set(_extra_flags "-fipa-pta")
+set(_extra_flags "-fmerge-all-constants -fipa-pta")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${_extra_flags}")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${_extra_flags}")
+
+# 128 KiB
+set(DEVILUTIONX_PALETTE_TRANSPARENCY_BLACK_16_LUT OFF)
+
+# set(DEVILUTIONX_RESAMPLER_SPEEX OFF)
+
+# Use lower resampling quality for FPS.
+set(DEFAULT_AUDIO_RESAMPLING_QUALITY 0)
+# Must use a tiny audio buffer due to RAM constraints.
+set(DEFAULT_AUDIO_BUFFER_SIZE 256)
+
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3 -g0")

```

Comment:
This should not be needed, stripping takes care of removing debug information and I think `-O3` is the default?